chainer/functions/noise/simplified_dropconnect.py

```diff
@@ -52,8 +53,12 @@ def check_type_forward(self, in_types):
     def forward(self, inputs):
         scale = inputs[1].dtype.type(1. / (1 - self.ratio))
         xp = cuda.get_array_module(*inputs)
-        mask_shape = (inputs[0].shape[0], inputs[1].shape[0],
-                      inputs[1].shape[1])
+        if self.batchwise_mask:
+            mask_shape = (inputs[0].shape[0], inputs[1].shape[0],
+                          inputs[1].shape[1])
+        else:
+            mask_shape = (inputs[1].shape[0], inputs[1].shape[1])
```

Comment:
We do not have to create `mask_shape` unless `self.mask` is `None`. So please move it after l.62